cmd/openshift-install/testdata/agent/configimage/manifests/default_manifests.txt

```diff
@@ -104,6 +104,8 @@ spec:
 status: {}
 -- expected/pull-secret.yaml --
 apiVersion: v1
+data:
+  .dockerconfigjson: ewogICJhdXRocyI6IHsKICAgICJxdWF5LmlvIjogewogICAgICAiYXV0aCI6ICJjM1Z3WlhJdGMyVmpjbVYwQ2c9PSIKICAgIH0KICB9Cn0=
```

Comment:
Regarding the integration tests, in order to match the issue reported inhttps://issues.redhat.com/browse/AGENT-422, my expectation would be to see a new one where:
* A set of ZTP manifests are provided
* The pull secret file contains the secret in the Data field
The expectation should check that the generated Secret file contains the expected StringData